File: src/main/intellibitz/sted/actions/ReOpenFontMapAction.java
package intellibitz.sted.actions;

import intellibitz.sted.STEDGUI;
import intellibitz.sted.ui.STEDWindow;
import intellibitz.sted.ui.TabDesktop;

import javax.swing.*;
import java.awt.event.ActionEvent;

public class ReOpenFontMapAction
        extends OpenFontMapAction {
    public ReOpenFontMapAction() {
        super();
    }

    public void actionPerformed(ActionEvent e) {
        final STEDWindow stedWindow = getSTEDWindow();
        STEDGUI.busy();
        final TabDesktop tabDesktop =
                stedWindow.getDesktop();
//        if (JOptionPane.CANCEL_OPTION != tabDesktop.saveDirty())
//        {
        final String fileName = (String) getValue(Action.NAME);
        tabDesktop.reopenFontMap(fileName);
//        }
        STEDGUI.relax();
    }

}

File: src/main/intellibitz/sted/actions/ReOpenFontMapAction.kt
package sted.actions

import sted.STEDGUI
import java.awt.event.ActionEvent

open class ReOpenFontMapAction : OpenFontMapAction() {
    override fun actionPerformed(e: ActionEvent) {
        STEDGUI.busy()
        val tabDesktop = stedWindow.desktop
        //        if (JOptionPane.CANCEL_OPTION != tabDesktop.saveDirty())
//        {
        val fileName = getValue(NAME) as String
        tabDesktop.reopenFontMap(fileName)
        //        }
        STEDGUI.relax()
    }
}